aws-lambda/src/main.rs

```diff
@@ -0,0 +1,51 @@
+use std::sync::Arc;
+
+use http::{to_request, to_response};
+use lambda_http::{run, service_fn, Error, Request, Response};
+use lazy_static::lazy_static;
+use runtime::init_runtime;
+use tailcall::async_graphql_hyper::GraphQLRequest;
+use tailcall::blueprint::Blueprint;
+use tailcall::config::reader::ConfigReader;
+use tailcall::http::{handle_request, AppContext};
+use tokio::sync::RwLock;
+
+mod http;
+mod runtime;
+
+lazy_static! {
+    static ref APP_CTX: RwLock<Option<Arc<AppContext>>> = RwLock::new(None);
+}
+
+async fn function_handler(event: Request) -> Result<Response<hyper::Body>, Error> {
+    let app_ctx = APP_CTX
+        .read()
+        .await
+        .clone()
+        .expect("AppContext not initialized yet, please wait");
+    let resp = handle_request::<GraphQLRequest>(to_request(event), app_ctx).await?;
+    Ok(to_response(resp)?)
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Error> {
+    tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::TRACE)
```

Comment:
by default it should be `WARN` 
and will be good to be able to specify it with env variable as it works with cli right now

Reply:
fixed the default. env variable is a bit complex because i need to add it to terraform instead. that's an issue for `tailcall-on-aws`